### DISCUSSION TITLE
O AutoBind não é encontrado.

### DISCUSSION THREAD

**Discussion by Neilom** [Flutterando/forum#131] [Perguntas e Respostas] [answered]
Estou com uma dúvida,  eu estava criando o projeto com o Flutter no channel Stable, mas eu vi que não foi mostrado o passo da factory do jeito antigo, então tentei mudar o meu Flutter para o Channel Beta, igual o do professor, porém o AutoBind não é encontrando e o projeto não consegue ser build

Teria alguma dica pra poder ajudar?

**Comment by eduardoflorence** [comment, marked as answer]
Olá @Neilom, o AutoBind não tem relação com a versão do Flutter. Ele pertence ao Flutter Modular, na versão 6, que ainda está em fase alpha.

```yaml
dependencies:
  flutter_modular: ^6.0.0-alpha.5
```
